packages/venia-ui/lib/components/Button/button.css

```diff
@@ -56,7 +56,6 @@
 .root_lowPriority {
     composes: root;
     border-width: 0;
-    color: rgb(var(--venia-error));
```

Comment:
This changes the forgot password button text from red to black. Luckily it's the only place, but this is a public API. Anyone using `Button` with `low` priority will see this change.

https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/SignIn/signIn.js#L66

Reply:
I've updated the signin component to set the color for this specific instance of the button. We worked with UX and determined all low priority buttons shouldn't have the error color.